src/GameWindow.cpp

```diff
@@ -391,6 +380,18 @@ void CGameWindow::OnKeyDown(const KeyEvent &ev)
     const bool acceptKey = true;
     if (acceptKey && g_CurrentScreen != nullptr && g_ScreenEffectManager.Mode == SEM_NONE)
     {
+        if (g_CtrlPressed && key == SDLK_v && g_EntryPointer != nullptr) // FIXME: OSX CMD Key

```

Comment:
The OSX Command key is SDLK_RGUI / SDLK_LGUI key or KMOD_LGUI / KMOD_RGUI.
To make Cmd act as Control on OSX, we'll need to wrap the GUI / CTRL modifiers / keys into a platform specific CTRL override via defines. This can be done another time though.

Reply:
yes because if we do that it can change the way how macro work ...

Reply:
Yes, but we must be consistent. If on OSX control is command, it should be everywhere or not?
It is weird as macros does not support CMD, only control. So maybe we should assume that CMD does not exists by itself and always make it work as control?

Reply:
yes I think the same way as you on UO CMD dont exist... but for a system command like paste it will

Reply:
Ok, so probably on OSX we'll force CMD -> CTRL. We can do in another fix. Will merge this.